### ISSUE TITLE
2.3 Biometric Data Collection for Background Investigations

### ISSUE THREAD

**Issue by wszwerc** [usnistgov/FIPS201#347]
### All Fields Are Required
**Organization Name (N/A, if individual)**: DHS

**Organization Type (see below for codes)**: 1 - Federal
 
**Reference (Include section/paragraph or pdf line number)**: Line 587-589
 
**Comment (Include rationale for comment)**: "2.3 Biometric Data Collection for Background Investigations

""These fingerprints MAY be taken from the full set of fingerprints collected in Section 2.3..."""

 
**Suggested Change**: """…MAY…"" should be ""…SHALL…"" to effectively maintain the chain of trust between background investigation and credentialing.

An exception should be allowed where ""Two fingerprints for OCC may be collected only after 1:1 biometric comparison of the applicant with the fingerprints collected in Section 2.3.""  This too maintains the chain of trust."

 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

**Comment by jimfenton** [comment]
See also #346

**Comment by hferraio** [comment]
closing this issue as it is a word-for-word duplicate of #346. Note: 346 has an additional suggestion at the end. Keeping #346 open.

**Comment by jglosx3** [comment]
Duplicate of issue #346.
